app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -337,6 +337,10 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi {
     }
   }
 
+  override fun showConfirmRemoveBloodSugarDialog(bloodSugarMeasurementUuid: UUID) {
+
```

Comment:
Please add a TODO to methods that you would add implementation to later.
This helps the reviewer know why the method body is empty.

Reply:
Oh. There is no implementation at all. I was expecting it in the later commits :)
Can you please add a TODO here?